spec/factories/school.rb

```diff
@@ -1,5 +1,16 @@
 # frozen_string_literal: true
 
+FactoryBot.define do
+  factory :local_authority do
+    code { Faker::Number.unique.decimal_part.to_s }
+    name { "Test local authority" }
+  end
+  factory :local_authority_district do
+    code { Faker::Number.unique.decimal_part.to_s }
+    name { "Test local authority district" }
+  end
+end
+
```

Comment:
Is there any reason these aren't in their own files? Pretty sure that's how the generator does it